src/components/rhelGraphCard/rhelGraphCard.js

```diff
@@ -12,13 +11,18 @@ import {
   Label as PfLabel
 } from '@patternfly/react-core';
 import { Skeleton, SkeletonSize } from '@redhat-cloud-services/frontend-components';
-import { Chart, ChartBar, ChartBaseTheme, ChartLabel, ChartStack, ChartTooltip } from '@patternfly/react-charts';
+import { Chart, ChartAxis, ChartBar, ChartStack, ChartTooltip } from '@patternfly/react-charts';
 import { connectTranslate, reduxActions } from '../../redux';
 import { helpers, dateHelpers, graphHelpers } from '../../common';
 import { rhelApiTypes } from '../../types/rhelApiTypes';
 
 class RhelGraphCard extends React.Component {
-  state = { dropdownIsOpen: false };
+  state = { dropdownIsOpen: false, width: 0 };
+
+  constructor(props) {
+    super(props);
+    this.containerRef = React.createRef();
+  }
```

Comment:
i am not sure, i'm used to setting this on the class itself instead of `state`. Also would change the test. It seems like best practice is to keep it there...
https://reactjs.org/docs/refs-and-the-dom.html#creating-refs

Reply:
oh - misread your comment... just declaring the class property the same way... updating now!